example/zping-server/zping-server.go

```diff
@@ -72,26 +73,29 @@ func main() {
 		file := *configPtr
 		configFile, err := config.NewFromFile(file)
 		if err != nil {
-			logger.Errorf("Problem loading configfile: %+v\n", err)
-			panic(err)
+			logrus.Errorf("Problem loading configfile: %+v\n", err)
```

Comment:
`os.Exit(1)`?

Reply:
If you don't use panic the logrus just print and continues